darwin/dataset/remote_dataset_v2.py

```diff
@@ -832,3 +856,18 @@ def register_multi_slotted(
                 print(f"  - {item}")
         print(f"Reistration complete. Check your items in the dataset: {self.slug}")
         return results
+
+    @retry(
+        wait=wait_exponential_jitter(initial=60, max=120),
+        stop=stop_after_attempt(5),
+        retry=retry_if_exception_type(requests.exceptions.RequestException),
+        before_sleep=before_sleep,
+    )
```

Comment:
I'll need some feedback on this. Currently, this basically says:

If we hit a 429, exponentially retry with a minimum wait time of 30s, and a maximum wait time of 240s. Introduce some [jitter](https://aws.amazon.com/builders-library/timeouts-retries-and-backoff-with-jitter/) into the wait times, and fail after a maximum of 5 retries.

Reply:
Great idea introducing some jitter :+1: For the values themselves, I would have a start of 60 seconds, ranging to 5 minutes with 10 retries. This may also not work ideally for all uses as it depends what concurrency this function is called.